src/ScottPlot5/ScottPlot5/Plot.cs

```diff
@@ -257,4 +258,14 @@ public void SaveImage(int width, int height, string path, int quality = 100)
     }
 
     #endregion
+
+    #region Legend
+    public IEnumerable<LegendItem> LegendItems()
```

Comment:
Does this need to be public? Interacting with items individually seems like pretty advanced stuff, so maybe users wishing to do this should just iterate the plottables and get the legend items themselves.

Reply:
I don't think it's trivial enough that it wouldn't be much of a loss if it was private (provided that you moved it to the `PlottableFactory` class so that it could access it), but I think there's a usecase for people who want to filter or split up the legend into multiple legends (for example by plot type).